src/Groups/gsets.jl

```diff
@@ -23,18 +24,162 @@ export
 # - conjugacy classes of group elements
 # - conjugacy classes of subgroups
 # - block system
-abstract type GSet{T} end
+abstract type GSet{TG,TE} end
```

Comment:
Personally I prefer slightly more "self-explanatory" parameter names:
```suggestion
abstract type GSet{GrpT,ElmT} end
```